js/data/item_data.js

```diff
@@ -354,3 +360,11 @@ function getNaturalGift(item) {
     }
     return {'t':'Normal','p':1};
 }
+function getTechnoBlast(item) {
```

Comment:
@Marty-D Oddly that was my thought process aswell but for what ever reason I went the other way with it